### PR TITLE
New brightness config

### PR DESCRIPTION
**Pull request by bebeli555**
this adds a Range config to the brightness setting. So people can change the brightness better as theres 49 different values between 1 and 50. It sets the brightness value by dividing the int value with 20 so its between 0.05 and 2.5 and changing it by one changes the brightness by 0.05. Previously the lowest was 0.5 and highest was 1.5 so people can now put it alot lower which can be cool to see the lighting better and higher to see better in caves and  stuff. #83

Also it converts the old brightness setting to the new one so the brightness config wont reset after updating. I also tested this alot and it all works fine.

### CODE REVIEW

**Comment by mithom** [comment]
how about making the in-game brightness slider sync with this value?

**Comment by bebeli555** [comment]
> how about making the in-game brightness slider sync with this value?
> because currently that slider is rendered useless

Yeah but i think its better this way as the plugins brightness doesn't work the same way as the in-game brightness does. Like the plugin has darker dungeons for example where in-game brightness never changes depending where you are so its going to be easier to toggle the plugin without needing to re adjust the in-game brightness. Also the in-game brightness still changes the brightness for item icons in inventory.

File: src/main/java/rs117/hd/HdPluginConfig.java [hunk 1]
```diff
@@ -162,17 +161,18 @@ default Contrast contrast()
 		return Contrast.DEFAULT;
 	}
 
+	@Range(
+		min = 1,
+		max = 50
+	)
 	@ConfigItem(
-		keyName = "brightness",
+		keyName = "brightness2",
```
Comment (kinztechcom) [inline src/main/java/rs117/hd/HdPluginConfig.java:169]: Maybe a better name here than `brightness2`, like `detailedBrightness` or something of the sort?

Comment (bebeli555) [reply src/main/java/rs117/hd/HdPluginConfig.java:169]: Yeah well i had to change it or otherwise it will override the old brightness setting and you cant convert it anymore. but the brightness2 only shows in the config file and the method to get it is still called brightness so it doesn't matter that much.

**Comment by yungando** [comment]
If this gets approved and added, can I request similar extensions to the saturation and contrast options?
